Configuration.props

```diff
@@ -9,19 +9,23 @@
     <HostOS Condition=" '$(HostOS)' == '' And '$(OS)' == 'Unix' And Exists ('/Applications') ">Darwin</HostOS>
     <HostOS Condition=" '$(HostOS)' == '' And '$(OS)' == 'Unix' ">Linux</HostOS>
     <HostCc Condition=" '$(HostCc)' == '' ">clang</HostCc>
-    <HostCxx Condition=" '$(HostCc)' == '' ">clang++</HostCxx>
+    <HostCxx Condition=" '$(HostCxx)' == '' ">clang++</HostCxx>
     <ManagedRuntime Condition=" '$(ManagedRuntime)' == '' And '$(OS)' != 'Windows_NT' ">mono</ManagedRuntime>
```

Comment:
I'd suggest setting the default compiler names to `HostCc = cc` and `HostCxx = c++` which on OSX are symlinked to `clang` and `clang++`, respectively, and on Linux they point to `g++` and `gcc`. This makes the setting more "portable" since clang isn't installed by default by most Linux distributions yet.